package.json

```diff
@@ -74,9 +75,8 @@
     "dotenv": "^8.2.0",
     "durable-functions": "^1.4.3",
     "express": "^4.15.3",
-    "fp-ts": "1.17.0",
+    "fp-ts": "1.17.4",
     "html-to-text": "^5.1.1",
-    "io-functions-commons": "^17.0.0",
     "io-functions-express": "^1.1.0",
     "io-ts": "1.8.5",
     "italia-ts-commons": "^8.5.0",
```

Comment:
Can we remove this package to use only `@pagopa/ts-commons`? Or we are planning this upgrade into another story/PR?

Reply:
Intentionally yes, let me see if we can in this same PR
@michaeldisaro thoughts?

Reply:
I wanted to do that too, but in another PR. If you agree we can manage to remove it in this PR. Anyway it's a good thing to get rid of the old dependency.